requirements/requirements-dev.txt

```diff
@@ -81,3 +81,5 @@ wolframalpha==3.0.1
 wrapt==1.11.1
 xmltodict==0.12.0
 yarl==1.3.0
+
+#
```

Comment:
@greysteil where do these lines come from?

Reply:
Oooh, a bug. It's because Dependabot is using the latest pip tools. I'll fix.

Reply:
Thanks for the quick fix!